package.json

```diff
@@ -36,7 +36,7 @@
     }
   ],
   "dependencies": {
-    "react-native-tabs": "^1.0.2"
+    "react-native-tab-navigator": "joenoon/react-native-tab-navigator#master"
```

Comment:
Are we still waiting on upstream changes on tab-navigator? Or can we point back to npm?